assets/src/edit-story/app/media/utils/getFileFromUrl.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { v4 as uuidv4 } from 'uuid';
+
+function getUrlExtension(url) {
+  return url.split(/[#?]/)[0].split('.').pop().trim();
+}
+
+async function getFileFromUrl(url, mimeType) {
```

Comment:
Nit: let's rename this to `fetch...`, e.g. `fetchRemoteFile`.

Makes it clearer that this does a remote fetch (just "`get`" could imply it's retrieved from elsewhere)